index.html

```diff
@@ -3,22 +3,219 @@
     <head>
         <meta charset="utf-8">
         <meta http-equiv="x-ua-compatible" content="ie=edge">
-        <title>Muli</title>
+        <title>Good-Call</title>
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
 
         <link rel="manifest" href="site.webmanifest">
         <link rel="apple-touch-icon" href="icon.png">
         <!-- Place favicon.ico in the root directory -->
 
-        <link rel="stylesheet" href="assets/css/styles.css">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.8.1/slick.min.css">
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.8.1/slick-theme.min.css">
+		<link rel="stylesheet" href="assets/css/styles.css">
     </head>
     <body>
-        
-        
+        <header class="header">
+			<div class="header__top" role="navigation">
+                <img class="header__good-call" alt="Good Call logo" src="assets/img/GoodCall_logo.svg">
+                <a class="header__hamburger" href="#"><img alt="hamburger icon for site navigation" src="assets/img/hamburger_icon.svg"></a>                        
+                <a class="header__login" href="#" data-reveal-id="login-modal">Login</a>
+            </div>        
+        </header>
 
-        <script src="node_modules/jquery/dist/jquery.min.js"></script>
-        <script src="node_modules/zurb-foundation-5/js/foundation/foundation.js"></script>
-        <script src="assets/js/project.js"></script>
+        <section class="section section--hero section--slider  text-center">
+            <div class="row">
+                <div class="columns large-8 large-offset-2 ">
+                    <div class="slides js-slider">
+                        <div class="slides__slide  section--slider__slide-1">
+                            <img class="section--hero__image" alt="classic college 2-story building" src="assets/img/hero-image-1.png">
```

Comment:
So this class would have a parent class of section, which means the class name would be `.section__hero-image`